lib/kernel/src/Makefile

```diff
@@ -119,6 +119,7 @@ MODULES = \
         logger \
         logger_backend \
         logger_config \
+	logger_handler \
```

Comment:
You must add `logger_handler` to `kernel.app.src`.

Reply:
Of course, fixed.